sensor/common/sensor/central_communication_impl.go

```diff
@@ -364,7 +367,7 @@ func (s *centralCommunicationImpl) initialPolicySync(stream central.SensorServic
 	if msg.GetPolicySync() == nil {
 		return errors.Errorf("second message received from Sensor was not a policy sync: %T", msg.Msg)
 	}
-	if err := detector.ProcessPolicySync(context.Background(), msg.GetPolicySync()); err != nil {
+	if err := detector.ProcessPolicySync(ctx, msg.GetPolicySync()); err != nil {
```

Comment:
I opted to pass down the parent context here rather than replace the `context.Background`.